### DISCUSSION TITLE
Integrate Microsoft SQL Server

### DISCUSSION THREAD

**Discussion by Wulfheart** [Budibase/budibase#906] [Help] [answered]
Hello,

is it possible to integrate Microsoft SQL Server? As far as I understand it is not supported yet. However, in enterprise IT, which is not in the "web dev space", Microsoft SQL Server is often used. Is there any possibility to contribute an integration for this if it is not supported yet?

**Comment by shogunpurple** [comment, marked as answer]
Hi @Wulfheart, thanks for raising this.

We have an implementation of our "included" external data sources coming very soon, and there will be the ability to contribute an integration to our open source integrations repository.

Watch this space for now, but until you can contribute directly, we'd love to hear more about your use case. What would the perfect integration experience look like?

Thanks!

**Reply by Wulfheart** [reply]
I’ve got a call scheduled with somebody from budibase. Is it ok for you if I explain it there?

**Reply by mjashanks** [reply]
Looking forward to it!
